src/states/LevelOne.js

```diff
@@ -1,31 +1,44 @@
-/* globals ButtonGroup, WorldMap */
+/* globals ButtonGroup, WorldMap, SceneGenerator */
 var LevelOne = function () {
   this.baseBtns = null;
   this.map = null;
   this.returnBtn = null;
+  this.sceneBtn = null;
+  this.scene = null;
 };
 
 LevelOne.prototype = {
 
   preload: function() {
     game.stage.backgroundColor = '#444444';
+    game.load.script('buttonGroup', 'src/ButtonGroup.js');
```

Comment:
This has already been loaded in Cutscene.js, and I think we should probably put the other two loads into Cutscene.js as well.